mitiq/_about.py

```diff
@@ -27,7 +27,10 @@
 
 import mitiq
 
-MITIQ_INSTALL_PATH = os.path.dirname(inspect.getsourcefile(mitiq))
+MITIQ_SOURCE_FILE = inspect.getsourcefile(mitiq)
+assert MITIQ_SOURCE_FILE
```

Comment:
Would this ever fail?

Reply:
`getsourcefile` has branches where it returns `None`, which is why mypy complained and wanted me to add something of this flavor